src/components/label/label.wp.scss

```diff
@@ -40,6 +40,11 @@ $label-wp-text-color-focused:         color($colors-wp, primary) !default;
   transform-origin: left top;
 }
 
+html[dir="rtl"] .label-wp[floating] {
+  transform: translate3d(-8px, 34px, 0);
```

Comment:
This rtl behavior is exactly the same as ltr behavior, so no need to write it again (this specific line)